presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -80,8 +80,8 @@ Properties controlling spilling
 
     Spilling works by offloading memory to disk. This process can allow some queries with large memory
     footprint to pass at the cost of slower execution times. Currently, spilling is supported only for
-    aggregations, so this property will not reduce memory usage required for joins, window functions and
-    sorting.
+    aggregations and joins, so this property will not reduce memory usage required for window functions
+    and sorting.
```

Comment:
or more precisely

"aggregations and joins (inner and left outer), so this property will not reduce memory usage required for window functions, sorting and other join types"